packages/o365/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.16.0"
+  changes:
+    - description: Populate `message` field from o365audit.Name instead of o365audit.Comments in SecurityComplianceAlerts events to better reflect Alert Titles.
```

```suggestion
    - description: Populate `message` field from the O364 Audit Log `Name` field instead of `Comments` in SecurityComplianceAlerts events to better reflect Alert Titles.
```
(`o365audit.*` is an implementation detail that the user never sees)